### DISCUSSION TITLE
How to style input.focus?

### DISCUSSION THREAD

**Discussion by jk0ehn** [neuefische/questions#238] [Web] [answered]
How style the focus that it go trough the icon?

![Bildschirmfoto 2022-10-10 um 14 43 18](https://user-images.githubusercontent.com/109575621/194869322-22993e9b-dd3f-4d96-9f2c-912e47a58f0a.png)

**Comment by pixelass** [comment, marked as answer]
You have to create a custom style.

Take a look at [focus-within](https://developer.mozilla.org/en-US/docs/Web/CSS/:focus-within) to style the parent element if a child is in focus

**Comment by jk0ehn** [comment]
THX
